### PR TITLE
OSDOCS-3713 RN: Azure Ultra SSD

### PR DESCRIPTION
**Pull request by bscott-rh**
Release notes for [CORS-1836](https://issues.redhat.com/browse/CORS-1836)
Doc preview: https://bscott-rh.github.io/openshift-docs/OSDOCS-3713-RN/release_notes/ocp-4-11-release-notes.html#ocp-4-11-azure-ultrassd
Doc PR: [OSDOCS-3713](https://issues.redhat.com/browse/OSDOCS-3713)

### CODE REVIEW

**Comment by bscott-rh** [comment]
@mgahagan73 @Amoghrd Hello Mike and Amogh, could one of you please take a look at this release note for CORS-1836? I will circle back and update the "Additional information" link with a valid URL once the main PR for the documentation is complete. Thank you

**Comment by Amoghrd** [comment]
Looks good!

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -100,6 +100,12 @@ For more information, see xref:../installing/installing_aws/installing-aws-secre
 
 //For more information, see ../installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc#installing-nutanix-installer-provisioned[Installing a cluster on Nutanix].
 
+[id="ocp-4-11-azure-ultrassd"]
```
Comment (EricPonvelle) [inline release_notes/ocp-4-11-release-notes.adoc:103]: Passive voice for "are installing"

Do you want to remove that URL? I take it being commented that you will uncomment this link when it goes live?

**Review by EricPonvelle — COMMENTED** [review]
Small nit

**Comment by mgahagan73** [comment]
Lookd good to me

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (bscott-rh) [reply release_notes/ocp-4-11-release-notes.adoc:103]: Thanks Eric - I believe I resolved the passive voice. I also uncommented and fixed the URL. The content that this release note addresses has not been merged yet, but the link and anchor are correct for when it is merged.

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**